### PR TITLE
Update footer and toolbar size and color

### PR DESCRIPTION
**Pull request by theyokohamalife**
Simplified the footer and toolbar a bit more for mobile. Will need changes to localization. We could set breakpoints to show different toolbar and footer text depending on the device. Most of our users are mobile though.

### Screenshots
Screen Shot 2021-06-24 at 1 40 21 AM (2)

Screen Shot 2021-06-24 at 1 40 11 AM (2)

### CODE REVIEW

File: components/SiteFooter.vue [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <template>
-  <v-footer fixed app dark color="accent">
-    <v-col fluid>
+  <v-footer fixed app dark color="footer">
+    <v-col fluid align-content="center" align="center">
       <v-row>
         <v-col>
           <v-icon>mdi-needle</v-icon>
-          {{ $t("footer.siteTitle", [new Date().getFullYear()]) }}
+          {{ "©" + new Date().getFullYear() }}
```
Comment (ann-kilzer) [inline components/SiteFooter.vue:7]: We can't copyright this is we want Netlify to approve us as open source

File: nuxt.config.js [hunk 2]
```diff
@@ -155,6 +155,8 @@ export default {
           warning: colors.amber.base,
           error: colors.deepOrange.accent4,
           success: colors.green.accent3,
+          toolbar: colors.indigo.lighten1,
```
Comment (ann-kilzer) [inline nuxt.config.js:158]: Oh I didn't know you could add more color vars, that's awesome

**Review by ann-kilzer — COMMENTED** [review]
I like the look here, but there are a lot of merge conflicts... Can we make sure to pull the latest main when adding changes?

Not sure how to proceed, we can either merge my changes into this branch, or re-apply these changes on a fresh branch.

File: components/SiteFooter.vue (hunk 1 above)
Comment (theyokohamalife) [reply components/SiteFooter.vue:7]: Ohh! Can we just remove that whole column then?

File: nuxt.config.js (hunk 2 above)
Comment (theyokohamalife) [reply nuxt.config.js:158]: Yup! It makes it much easier to change the overall theme.

**Comment by theyokohamalife** [comment]
> I like the look here, but there are a lot of merge conflicts... Can we make sure to pull the latest main when adding changes?
> 
> Not sure how to proceed, we can either merge my changes into this branch, or re-apply these changes on a fresh branch.

I didn't see any conflicts, but I rebased off main just to be sure. Feel free to merge into this branch or use the changes in another one. Whatever is easiest.

File: components/SiteFooter.vue [hunk 3]
```diff
@@ -32,9 +34,13 @@
...
 
-<style>
+<style scoped>
 a:link,
 a:visited {
   color: white;
+  text-decoration: none;
+}
+.col {
+  padding: 6px !important;
```
Comment (ann-kilzer) [inline components/SiteFooter.vue:44]: In the future, we might want to move away from `!important` in CSS, but tonight, let's just get this working.

**Review by ann-kilzer — APPROVED** [review]
Please confirm the Netlify build, but I really like this look. 👏🏻 

LGTM

File: components/SiteFooter.vue (hunk 3 above)
Comment (theyokohamalife) [reply components/SiteFooter.vue:40]: Sorry this PR is full of CSS habits that I know @Ermish won't like either lol 😭

Comment (Ermish) [reply components/SiteFooter.vue:38]: 😱 😱 😱

**Pull request merged**